internal/rootcoord/step.go

```diff
@@ -251,10 +251,12 @@ func (s *releaseCollectionStep) Weight() stepPriority {
 type dropIndexStep struct {
 	baseStep
 	collID UniqueID
+	// partID = -1 means to drop the index at the collection level
+	partID UniqueID
```

Comment:
`step` is just a wrapper of broker api. So just keep partIDs same with broker api here.